tests/projects/project1/packages/hello-go/Cargo.toml

```
@@ -1,12 +1,12 @@
 [package]
-name = "hello-go"
+name = "pkg-a"
```

This file seems like it should be deleted also, or if not:
```suggestion
name = "hello-go"
```

---

Errant change.